README.md

```diff
@@ -92,381 +92,172 @@
-
-n_training_examples = 2000
-noise = ivy.random.random_normal(shape=(n_training_examples, 1), mean=0, std=0.1)
-x = ivy.linspace(-6, 3, n_training_examples).reshape((n_training_examples, 1))
-y = 0.2 * x ** 2 + 0.5 * x + 0.1 + noise
+- [How to Convert your code between frameworks?](https://unify.ai/docs/ivy/demos/learn_the_basics/04_transpile_code.html) 
+- [How to write a framework-agnostic code?](https://unify.ai/docs/ivy/demos/learn_the_basics/01_write_ivy_code.html) 
+- Accelerate your development (WIP)
+- Autotune and optimize models (WIP) 
 
+------------------------------------------------------------------------
+### Ivy in 3 steps
 
-def loss_fn(v, x, target):
-    pred = model(x, v=v)
-    return ivy.mean((pred - target) ** 2)
+You can find quite a lot more examples in the corresponding section below, but using Ivy is as simple as:
+1. <details>
+    <summary><b>Install</b></summary>
 
-for epoch in range(40):
-    # forward pass
-    pred = model(x)
+    There are various ways to use Ivy, depending on your preferred
+    environment:
 
-    # compute loss and gradients
-    loss, grads = ivy.execute_with_gradients(lambda params: loss_fn(*params), (model.v, x, y))
+    ### Installing using pip
 
-    # update parameters
-    model.v = optimizer.step(model.v, grads)
+    The easiest way to set up Ivy is to install it using pip with the
+    following command:
 
-    # print current loss
-    print(f'Epoch: {epoch + 1:2d} --- Loss: {ivy.to_numpy(loss).item():.5f}')
+    ``` bash
+    pip install ivy
+    ```
 
-print('Finished training!')
-```
+    or alternatively:
 
-The model\'s output can be visualized as follows:
+    ``` bash
+    python3 -m pip install ivy
+    ```
 
-<div align="center">
-   <img width="50%" class="dark-light" src="https://raw.githubusercontent.com/unifyai/unifyai.github.io/main/img/regressor_lq.gif">
-</div>
+    <details>
+    <summary>Docker</summary>
 
+    If you prefer to use containers, we also have pre-built Docker images
+    with all the supported frameworks and some relevant packages already
+    installed, which you can pull from:
 
-As always, you can find more information about [Ivy as a framework in
-the
-docs!](https://unify.ai/docs/ivy/overview/design/ivy_as_a_framework.html)
+    ``` bash
+    docker pull unifyai/ivy:latest
+    ```
 
-### When should I use Ivy as a framework?
+    If you are working on a GPU device, you can pull from:
 
-As Ivy supports multiple backends, writing code in Ivy breaks you free
-from framework limitations. If you want to publish highly flexible code
-for everyone to use, independently of the framework they are using, or
-you plan to develop ML-related tools and want them to be interoperable
-with not only the already existing frameworks, but also with future
-frameworks, then Ivy is for you!
+    ``` bash
+    docker pull unifyai/ivy:latest-gpu
+    ```
+    </details>
 
-## Setting up Ivy
+    <details>
+    <summary>from source</summary>
```

Comment:
caps in subtitle, would be good to go through the text after doing changes to ensure spelling and style is consistent :smile: